deblur/workflow.py

```diff
@@ -41,9 +47,95 @@ def dereplicate_seqs(seqs_fp,
         log_name=log_name)
 
 
-def remove_artifacts_seqs(seqs_fp):
-    """Step 3: remove artifacts from FASTA file"""
-    pass
+def remove_artifacts_seqs(seqs_fp,
+                          ref_fp,
+                          output_fp,
+                          ref_db_fp=None,
+                          negate=False,
+                          threads=1):
+    """Step 3: remove artifacts from FASTA file using
```

Comment:
Remove `Step 3` from here. This function should be agnostic of the overall workflow. The idea of step is introduced in the workflow function that will execute everything not here. Also it will helps to keep this first description in a single line (otherwise it doesn't render correctly in the numpy standard)